autoload/parinfer.vim

```diff
@@ -52,6 +53,8 @@ function! parinfer#init() abort
 
     lua parinfer.enter_buffer()
 
```

Comment:
@gpanders  also need a `let b:parinfer_enabled = 1` before the `doautocmd`, then it works